uwp/xbox-apps/devkit-deactivation.md

```diff
@@ -1,43 +1,43 @@
 ---
-title: Xbox One Developer Mode deactivation
+title: Xbox Developer Mode deactivation
 description: If you decide you no longer want to use your console for development, use these steps to deactivate Developer Mode.
-ms.date: 02/08/2017
+ms.date: 09/23/2023
 ms.topic: article
 keywords: windows 10, uwp
 ms.assetid: 244124dd-d80a-4a72-91db-1c9c2fbc7c3c
 ms.localizationpriority: medium
 ---
-# Xbox One Developer Mode deactivation
+# Xbox Developer Mode deactivation
 
 If you decide you no longer want to use your console for development, use the following steps to deactivate Developer Mode.
 
 ## Switch to Retail Mode
 
-First, return your Xbox One console to Retail Mode.
+First, return your Xbox console to Retail Mode.
 
 1. Open **Dev Home**.
 
-2. Select **Leave Dev Mode**.  Your console will restart in Retail Mode.  
+2. Select **Leave Dev Mode**. Your console will restart in Retail Mode.
 
    ![Leaving Developer Mode](images/devkit-deactivation-1.png)
 
 Now deactivate your console by using one of the following methods.
 
-## Deactivate your console using the Dev Mode Activation app
+## Deactivate your console using the Xbox Dev Mode app
 
-The preferred method of deactivating Developer Mode on your console is to use the **Dev Mode Activation** app. 
+The preferred method of deactivating Developer Mode on your console is to use the [Xbox Dev Mode](https://www.microsoft.com/en-us/p/xbox-dev-mode/9nljhzjrn0f4) app. 
 
-1. Navigate to **Games & apps** > **Apps**.
+1. Navigate to **My Games & apps** > **Apps**.
   
-   ![Activation Step 3](images/devkit-deactivation-5.png)    
+   ![Activation Step 3](images/devkit-deactivation-5.png)
    
-2.  Open the Dev Mode Activation app.
+2.  Open the Xbox Dev Mode app.
 
 3.  Select **Deactivate**.
   
     ![Deactivate console](images/deactivation-app.png)
 
-See [Xbox One Developer Mode activation](devkit-activation.md) for more information about the **Dev Mode Activation** app. 
+See [Xbox Developer Mode activation](devkit-activation.md) for more information about the [Xbox Dev Mode](https://www.microsoft.com/en-us/p/xbox-dev-mode/9nljhzjrn0f4) app. 
```

Comment:
Same thing here - remove en-us from the Xbox Dev Mode Store link

Reply:
Fixed.